clients/js/spec/zawgyi_converter_spec.js

```diff
@@ -634,12 +634,37 @@ var zawygi_unicode_convert_data = [
 // building this file before Jasmine can run it.
 
 var ZawgyiConverter;
+var CompatTestZ2Usource, CompatTestZ2Uoutput, CompatTestU2Zsource, CompatTestU2Zoutput;
```

Comment:
Classes like `ZawgyiConverter` start with an uppercase letter.  Variables like these should start with lowercase letters, like `compatTestZ2Usource`

Reply:
Done